simulator.py

```diff
@@ -72,7 +72,12 @@ def __init__(self, env, name, age, rng, infection_timestamp, household, workplac
         else:
             self.carefulness = (round(self.rng.normal(25, 10)) + self.age/2) / 100
 
+        # allergies
+        self.has_allergies = self.rng.rand() < P_ALLERGIES
+        len_allergies = self.rng.normal(1/self.carefulness, 1)
+        self.len_allergies = 1 if len_allergies < 1 else round(len_allergies)
```

Comment:
Do we need allergies?? Is it relevant for covid?

Reply:
It's like cold/flu; another source of symptoms. I thought it might be sort of important/useful to have one of the sources be totally random / not communicable.

Reply:
Yeah! I think it's good to have them. really impressive you have nailed down all these parameters. 💯